src/PAModel/Serializers/SourceSerializer.cs

```diff
@@ -251,6 +251,15 @@ public static CanvasDocument LoadFromSource(string directory2, ErrorContainer er
             LoadDataSources(app, dir, errors);
             LoadSourceFiles(app, dir, templateDefaults, errors);
 
+            foreach (var file in dir.EnumerateFiles(PackagesDir))
```

Comment:
Minor: we load other items from packages dir up on line 160, consider specifically looking for this file there, instead of looping all the files in that directory.